wolfssh/ssh.h

```diff
@@ -247,6 +247,7 @@ enum WS_FormatTypes {
     WOLFSSH_FORMAT_PEM,
     WOLFSSH_FORMAT_RAW,
     WOLFSSH_FORMAT_SSH,
+    WOLFSSH_FORMAT_TPM,
```

Comment:
This WOLFSSH_FORMAT_TPM is not used in the client example. Did you plan to make an option to load the key from TPM via wolfSSH_ReadKey_* function ?

Reply:
This is a leftover from one of the previous tries (code iterations). Thanks for catching it.

Indeed, this was its use. I think this was variant 2 of the integration.